packages/terra-embedded-content-consumer/docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Terra Embedded Content Consumer
+
+The Embedded Content Consumer is the application component which is embedding content within it.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-embedded-content-consumer`
+  - `yarn add terra-embedded-content-consumer`
+
+## Usage
+
+```jsx
+import React from 'react';
+import EmbeddedContentConsumer from 'terra-embedded-content-consumer';
+
+<EmbeddedContentConsumer
+  src="https://google.com"
+  onLaunch={() => { console.log('launched'); }}
+  onAuthorize={() => { console.log('Authorized'); }}
+  onFullScreen={() => { window.open('https://yahoo.com'); }}
```

Comment:
I think we should remove full-screen hook or at least deprecate it or hide it from doc. Since full-screen was first coded, we have shifted our strategy away from discrete API calls and towards using a generic _trigger_ api call to pass in the event desired. 

Removing this hook will streamline the API so that there are no first-lass event handlers outside of the life-cycle events.